src/utils/processDifferences.js

```diff
@@ -1,21 +1,28 @@
 import CONFIG from "../constants/config";
 
 const processDifferences = ({ differenceInformation }) => {
+  const titleOfChanges = [];
+  const detailOfChanges = [];
+
   if (
     differenceInformation === CONFIG.NEW_NODE ||
     differenceInformation === CONFIG.NEW_FRAME
   ) {
-    return { text: differenceInformation, className: "active" };
+    return {
+      titleOfChanges: ["NEW✨"],
+      detailOfChanges: [differenceInformation],
+      className: "active",
+    };
   }
 
   const modifiedInformation = JSON.parse(differenceInformation);
-  let differenceTexts = "";
 
   for (const key in modifiedInformation) {
-    differenceTexts += `${key}(변경)\n${modifiedInformation[key]}\n`;
+    titleOfChanges.push(`${key}`);
```

Comment:
- `P1`
```suggestion
    titleOfChanges.push(key);
```
문자열과 동적변수가 합쳐지는게 아니라서 백틱은 제거하셔도 될 것 같습니다!

\+
`for...in`을 사용할 때 `isOwnProperty`모듈을 사용하면 좋을 것 같습니다